portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+
+/** A comment on the portfolio */
+public final class Comment {
+
+  private final String name;
+  private final String comment;
+  private final long timestamp;
+
+  public Comment(String name, String comment, long timestamp) {
+    this.name = name;
+    this.comment = comment;
+    this.timestamp = timestamp;
+  }
+}
```

Comment:
Please add a new line at the end of file. It's always a good practice and no new line will cause errors in C

Reply:
Fixed now. Thank you for the heads up.